doc/release-notes-19093.md

```diff
@@ -0,0 +1,7 @@
+RPC changes
+------------
+
+### 'testmempoolaccept` improvement
+
+`testmempoolaccept` now additionally returns the transaction fee in the result.
+Fee is only returned if transaction is accepted in the mempool.
```

Comment:
"Fee is only returned if transaction would be accepted to the mempool." is more accurate (testmempoolaccept doesn't actually accept transactions to the mempool).

Reply:
updated.